tests/manage/z_cluster/test_drain_node_mon.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.resources.pod import get_mon_pods, get_pod_obj
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes
+from ocs_ci.helpers.helpers import verify_pdb_mon
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier4a,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier4a
```

Comment:
Please also create a test case in polarion if not done already and add the id .
@pytest.mark.polarion_id("<polarionid>")

Reply:
done